eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentLifecycle.scala

```diff
@@ -208,7 +209,8 @@ object PaymentLifecycle {
                          assistedRoutes: Seq[Seq[ExtraHop]] = Nil,
                          finalCltvExpiry: Long = Channel.MIN_CLTV_EXPIRY,
                          maxAttempts: Int,
-                         routeParams: Option[RouteParams] = None) extends GenericSendPayment {
+                         routeParams: Option[RouteParams] = None,
+                         paymentRequest_opt: Option[PaymentRequest] = None) extends GenericSendPayment {
```

Comment:
I think that the `paymentRequest_opt` attribute should be moved upward in the constructor arguments list, so that it's alongside `paymentHash`, `targetNodeId`, and not alongside payment parameters arguments.

```scala
case class SendPayment(amountMsat: Long,
                         paymentHash: ByteVector32,
                         targetNodeId: PublicKey,
                         paymentRequest_opt: Option[PaymentRequest] = None,
                         assistedRoutes: Seq[Seq[ExtraHop]] = Nil,
                         finalCltvExpiry: Long = Channel.MIN_CLTV_EXPIRY,
                         maxAttempts: Int,
                         routeParams: Option[RouteParams] = None) extends ...
```

Reply:
Okay, reorganized in 7c2e8c9